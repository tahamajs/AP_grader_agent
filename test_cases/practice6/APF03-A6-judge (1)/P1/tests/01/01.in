POST login ? username "low_mist" password "meoow"
POST signup ? username "low_mist" password "meow"
POST signup ? username "amir_akbar" password "1382"
POST logout ?
POST login ? username "low_mist" password "meoow"
POST login ? username "low_mist" password "meow"
POST logout ?
POST login ? username "low_mist" password "meoow"