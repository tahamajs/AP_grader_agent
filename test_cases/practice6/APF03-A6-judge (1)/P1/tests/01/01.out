Not Found
OK
Permission Denied
OK
Permission Denied
OK
OK
Permission Denied